fix: state init in angular in correct sequence (#1520)
BuilderIO/mitosis: .changeset/rude-garlics-impress.md

---
'@builder.io/mitosis': patch
---

> Fix: state initialization sequence in angular, init local states first so that bindings that depend upon them can use the correct values on ngOnInit

Reviewer: ```suggestion
Angular: Fix: state initialization sequence. Initialize states in `ngOnInit` first, followed by bindings that depend upon them.
```
